float32[] input_pose
---
float32[] ee_pose